Gemfile

```diff
@@ -113,6 +112,8 @@ source 'https://rails-assets.org' do
   gem 'rails-assets-c3', '~> 0.4.10'
   gem 'rails-assets-bootstrap-select', '~> 1.5.4'
   gem 'rails-assets-bootstrap-hover-dropdown', '~> 2.0.11'
+  gem 'rails-assets-angular', '~> 1.4.3'
+  gem 'rails-assets-angular-mocks', '~> 1.4.3'
```

Comment:
yes - I see no point in holding up merging an important PR like this that @abonas needs because of sorting gems for readability, which can be done in a separate PR